### DISCUSSION TITLE
Express JS server Deploy to Vercel?

### DISCUSSION THREAD

**Discussion by csdan2019** [vercel/community#6318] [Help] [answered]
### Question

Hi,

I have a react app that communicates to an express js.  Is it possible to deploy the express js that only contains server information from a github repository independently and then deploy my frontend to vercel?

**Comment by amyegan** [comment, marked as answer]
Hey @csdan2019. Yes, it's possible to deploy a frontend to Vercel use it with a backend service hosted elsewhere.
